plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchService.groovy

```diff
@@ -690,7 +690,7 @@ class AzBatchService implements Closeable {
                 .withFilePath('azcopy')
 
         def poolStartTask = new StartTask()
-                .withCommandLine('bash -c "chmod +x azcopy && mkdir \$AZ_BATCH_NODE_SHARED_DIR/bin/ && cp azcopy \$AZ_BATCH_NODE_SHARED_DIR/bin/" ')
+                .withCommandLine(startTaskCmd(spec.opts))
                 .withResourceFiles(resourceFiles)
```

Comment:
Wonder if the `startTask` should be added to the default one instead of overriding it

Reply:
I think it makes sense to allow the customisation of the `startTask` script, however the control over the azcopy should be left to nextflow. 

I've added an option to disable explicitly or implicitly when fusion is enabled. See here 

https://github.com/nextflow-io/nextflow/pull/4883

Reply:
Kind of, but it should be taken in consideration that that now the azcopy command can be absent. See here 

https://github.com/nextflow-io/nextflow/blob/74d7d7a8c58e40b19d8d10b0af4f0b59c9524eb2/plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchService.groovy#L685-L695